BLSQ/iaso
# IA-2853: public registry front part 1, the plugin & the home page

Having a landing page for a public registry pointing to a public version of the registry. This PR is focusing on plugin implementation, not the integration of the registry

Related JIRA tickets : IA-2853

## Self proofreading checklist


## Doc
-

## Changes

- added a ne plugin registry
- tweak a bit the existing plugin config to allow custom urls and app_name, api should also be optional
- fixing png images in the build with asset-module

## How to test

You need to enable `regsitry` plugin  in your envs in order to see it

After you can visit it on http://localhost:8081/registry/home, without being logged. Check that logo are working.


## Print screen / video

https://github.com/BLSQ/iaso/assets/12494624/d8f608d2-0eb3-47b0-8c29-6573c3fcaa89
